swarm/network/simulations/discovery/dynamic_test.go

```diff
@@ -0,0 +1,555 @@
+package discovery
```

Comment:
this is great for now for this PR abut i want to replace this test with one using janos's dynamic framework with a global health check